continuous_integration/azure/install.sh

```diff
@@ -13,6 +13,9 @@ if [ "$USE_CONDA" = "yes" ]; then
     # Install main dependencies
     conda install -q -y -c spyder-ide --file requirements/conda.txt
 
+    # Install dependencies not in conda
+    pip install -q applaunchservices
```

Comment:
Please remove this and add `applaunchservices` to `requirements/conda.txt`. It'll be installed for all OSes but that's not so serious because it's not used on Linux and Windows.